x/margin/keeper/keeper.go

```diff
@@ -293,13 +293,15 @@ func (k Keeper) CalculatePoolHealth(pool *clptypes.Pool) sdk.Dec {
 	ExternalLiabilities := sdk.NewDecFromBigInt(pool.ExternalLiabilities.BigInt())
 	NativeAssetBalance := sdk.NewDecFromBigInt(pool.NativeAssetBalance.BigInt())
 	NativeLiabilities := sdk.NewDecFromBigInt(pool.NativeLiabilities.BigInt())
+	UnsettledExternalLiabilities := sdk.NewDecFromBigInt(pool.UnsettledExternalLiabilities.BigInt())
```

Comment:
This line is panicking in the test. What would prevent this from happening in prod?

Reply:
weird, tests were passing last I ran them. 
looks like it's because the field isn't initialized. might have to initialize them all to 0 in migrator